changelog/8970.improvement.md

```diff
@@ -0,0 +1 @@
+Adapt `rasa test` to sort `IntentTEDPolicy` warnings by severity.
```

Comment:
is this behavior documented in the docs @dakshvar22 ?

Reply:
Yes, it is.